apps/passport-client/components/screens/ScannedTicketScreens/PodboxScannedTicketScreen/sections/actions/checkin/CheckInActionSection.tsx

```diff
@@ -71,24 +76,35 @@ export function CheckInActionSection({
   }
 
   if (executor.result?.success) {
+    const successMessage = precheck?.value?.actionScreenConfig
+      ?.actionSuccessCopy
+      ? precheck.value.actionScreenConfig.actionSuccessCopy
+      : "Checked In";
     return (
       <>
-        <StatusContainer size="small">Checked In</StatusContainer>
+        <StatusContainer size="small">{successMessage}</StatusContainer>
       </>
     );
   } else if (executor.result?.error) {
+    const errorMessage = precheck?.value?.actionScreenConfig?.actionErrorCopy
+      ? precheck.value.actionScreenConfig.actionErrorCopy
+      : "Checked In";
```

Comment:
I think this should say "Not Checked In", since this is an error state.

Reply:
https://github.com/proofcarryingdata/zupass/pull/1697/commits/9df74543aa2285df7dc9b35ae6485fcc5954bffd